src/Tickets/Order_Modifiers/Table_Views/Order_Modifier_Table.php

```diff
@@ -256,10 +261,15 @@ public function search_box( $text, $input_id, $placeholder = '' ) {
 	 * @return void
 	 */
 	public function render_tabs(): void {
-		$modifiers = tribe( Controller::class )->get_modifiers();
+		$modifiers = $this->get_modifiers();
 
-		$current_modifier = tribe_get_request_var( 'modifier', 'coupon' );
+		// If we don't have multiple modifiers, don't render tabs.
+		if ( count( $modifiers ) < 2 ) {
+			return;
+		}
```

Comment:
Based off of the comment, shouldn't this be `1` and not `2`?
```suggestion
		// If we don't have multiple modifiers, don't render tabs.
		if ( count( $modifiers ) < 1 ) {
			return;
		}
```